app/models/calculation/Calculation.scala

```diff
@@ -86,6 +86,36 @@ object SavingsAndGains {
   implicit val writes: OWrites[SavingsAndGains] = Json.writes[SavingsAndGains]
 }
 
+case class ReductionsAndCharges(giftAidTax: Option[BigDecimal] = None,
+                                totalPensionSavingsTaxCharges: Option[BigDecimal] = None,
+                                statePensionLumpSumCharges: Option[BigDecimal] = None,
+                                totalStudentLoansRepaymentAmount: Option[BigDecimal] = None,
+                                propertyFinanceRelief: Option[BigDecimal] = None,
+                                incomeTaxDueAfterTaxReductions: Option[BigDecimal] = None,
+                                totalIncomeTaxDue: Option[BigDecimal] = None
+                               )
+
+object ReductionsAndCharges{
+  implicit val reads: Reads[ReductionsAndCharges] = (
+    readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \ "detail" \ "incomeTax" \ "giftAid" \ "giftAidTax") and
+      readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \"summary" \ "incomeTax"  \ "totalPensionSavingsTaxCharges") and
+      readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \"summary" \ "incomeTax"  \ "statePensionLumpSumCharges") and
+      readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \"summary" \ "totalStudentLoansRepaymentAmount") and
+      readNullable[BigDecimal](__ \ "allowancesDeductionsAndReliefs" \ "detail" \ "reliefs" \ "residentialFinanceCosts" \ "propertyFinanceRelief") and
+      readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \"summary" \ "incomeTax" \ "incomeTaxDueAfterTaxReductions") and
+      readNullable[BigDecimal](__ \ "incomeTaxAndNicsCalculated" \"summary" \ "incomeTax" \ "totalIncomeTaxDue")
```

Comment:
We are not displaying this about in view which I believe according to prototype is section **Income Tax due after additional charges**

Reply:
incomeTaxDueAfterTaxReductions is not listed as a requirement in this story. But have added it to the model to be used later.